src/sentry/search/events/fields.py

```diff
@@ -1417,7 +1417,9 @@ def is_accessible(self, acl=None):
             calculated_args=[
                 {
                     "name": "tolerated",
-                    "fn": lambda args: args["satisfaction"] * 4.0 if args["satisfaction"] else None,
+                    "fn": lambda args: args["satisfaction"] * 4.0
+                    if args["satisfaction"] is not None
```

Comment:
💀  this sent me on a short wild goose chase

Reply:
What about the `apdex` function? I believe it needs the same changes?

Reply:
We don't calculate "tolerated" on apdex

Reply:
([snuba](https://github.com/getsentry/snuba/blob/41d7fe76aaf8a594e8f6e84015607dcde3f67ad4/snuba/query/processors/performance_expressions.py#L18) does it)

Reply:
:+1:, can we add a test for apdex too though?